src/app/workflow/view/view.service.ts

```diff
@@ -134,8 +168,11 @@ export class ViewService {
     if (selectedWorkflow.id === workflow.id) {
       this.workflowService.setWorkflow(workflow);
     }
+
+    const workflowName: string = workflow.workflowName || workflow.repository;
```

Comment:
I'm wondering if it should be either repository + '/' + workflowName or repository depending on if the name is present. Not sure if that takes up too much space in the confirm message.

Reply:
Added a note to https://github.com/dockstore/dockstore/issues/2846